00 Base/src/redux/reducers/Mesages.Reducer.test.js

```diff
@@ -0,0 +1,41 @@
+import MessagesReducer from './MessagesReducer';
+import { GET_ALL, ADD_MESSAGE } from '../actions/MessagesActions'
+import expect from 'expect';
+
+
+const getAllAction = {
+    type: GET_ALL,
+    payload: [
+        {
+            id: 0,
+            subject: 'Hello world',
+            body: 'Hello world'
+        }
+    ]
+}
+
+const addMessageAction = {
+    type: ADD_MESSAGE,
+    payload: {
+            id: 1,
+            subject: 'Hello world again',
+            body: 'Hello world again'
+        }
+}
+
+describe('Messages Reducer', () =>{
+    it('should return initial state', () =>{
+        expect(MessagesReducer(undefined, {})).toEqual({messages: []})
+    })
+
+    it('should save in the state GET_ALL action data', () =>{
+        expect(MessagesReducer({}, getAllAction)).toEqual({messages: getAllAction.payload});
+    })
+    
+    it('should add a new message with ADD_MESSAGE data', () =>{
+        const messages = getAllAction.payload;
+        expect(MessagesReducer({messages: messages}, addMessageAction ).messages).toHaveLength(2);
+    })
+})
+
+
```

Comment:
Did you check the total covarege? We need at least 75%

Reply:
I checked coverage per file, but I thought 75% total coverage was only for the third task. I will increase it.

Reply:
Ok, thats nice